$NetBSD$

Seems to fail because of compiler wrappers.
pkgsrc is responsible to provide the compiler, do not check here.

--- configure.orig	2023-09-24 17:08:59.000000000 +0000
+++ configure
@@ -213,7 +213,7 @@ echo "done, using '$CMD_INSTALL_USEROPT 
 
 ############# OS INDEPENDENT PART #############
 
-out "checking for gcc..."; $COMPILER -v >/dev/null 2>&1; check "yes" "no" "You need gcc (or set PATH)."
+#out "checking for gcc..."; $COMPILER -v >/dev/null 2>&1; check "yes" "no" "You need gcc (or set PATH)."
 
 ###############
 out "checking for libc..."
